app/controllers/api/v1/data_studio/school_rollout_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module DataStudio
+      class SchoolRolloutController < Api::ApiController
+        include ApiTokenAuthenticatable
+        include Pagy::Backend
+
+        def index
+          render json: ::DataStudio::SchoolRolloutSerializer.new(paginate(rollout_data))
+            .serializable_hash.to_json
+        end
+
+      private
+
+        def rollout_data
+          School
+            .eligible
+            .joins(:nomination_emails)
```

Comment:
Once we start updating schools from GIAS, we will have eligible schools who have not been sent a nomination email

Reply:
Thanks, good to know so that needs to be a left outer join then